packages/elements/src/components/Modal/modal.stories.tsx

```diff
@@ -12,7 +12,7 @@ const BasicUsage = ({ tapOutsideToDissmiss = true }) => {
 
   return (
     <PortalProvider>
-      <section className="section">
+      <section className="section px-4 py-4">
```

Comment:
You will get the padding for free with the section component and the rendered HTML will look better in storybook